ci/mac_ci_steps.sh

```diff
@@ -42,16 +42,21 @@ fi
 # is somewhat more deterministic (rather than interleaving the build
 # and test steps).
 
+DEFAULT_TEST_TARGETS=(
+  "//test/integration:integration_test"
+  "//test/integration:protocol_integration_test"
+  "//test/integration:tcp_proxy_integration_test")
+
 if [[ $# -gt 0 ]]; then
-  TEST_TARGETS=$*
+  TEST_TARGETS=("$*")
```

Comment:
i think you can just do `TEST_TARGETS=$@` here - its already an array - this converts it to a string and back again

Reply:
I tried and actually neither work, it seems it has to be `("$@")`. Yay bash. :)